centralstore/redis_store.go

```diff
@@ -39,15 +39,27 @@ type RedisBasicStoreOptions struct {
 var _ BasicStorer = (*RedisBasicStore)(nil)
 
 func NewRedisBasicStore(opt RedisBasicStoreOptions) *RedisBasicStore {
-	var host string
-	if opt.Host == "" {
-		host = "localhost:6379"
+	host := "localhost:6379"
```

Comment:
This pattern of setting defaults in the New() function is probably not the way we want to do it -- we want the defaults set by the client, not by the BasicStore. Ideally they're set in the config package, but you could, for now, put them in RedisBasicStoreOptions (which will move to the config package eventually).